Update section 8 to be more flexible about regular meeting days

tim-schilling opened the pull request:
Our process tends to be more receptive to directors' availability for meetings and is less rigid than the bylaws indicate.

tim-schilling committed:
Update section 8 to be more flexible about regular meeting days

Our process tends to be more receptive to directors' availability for meetings and is less rigid than the bylaws indicate.

drewbrew commented on bylaws.md:
```diff
@@ -88,7 +88,7 @@ c) The corporation adopts and implements some means of verifying (1) that all pe
 ### SECTION 8. REGULAR AND ANNUAL MEETINGS ###
 
-Regular meetings of directors shall be held on the first Wednesday of each month at 2:00 PM, unless such day falls on a legal holiday, in which event the regular meeting shall be held at the same hour and place on the next business day.
+Regular meetings of directors shall be held on the first Wednesday of each month at 2:00 PM, unless such day falls on a legal holiday or a signficant number of directors are unable to attend, in which event the regular meeting shall be held at the same hour and place on the next business day or another generally agreed upon day.
```
```suggestion
Regular meetings of directors shall be held on the first Wednesday of each month at 2:00 PM Central Time, unless such day falls on a legal holiday or a significant number of directors are unable to attend, in which event the regular meeting shall be held at the same hour and place on the next business day or another generally agreed upon day.
```

drewbrew replied:
Spell check plus clarifying the time zone

tim-schilling committed:
Spell check plus clarifying the time zone

Co-authored-by: Drew Winstel <drewbrew@users.noreply.github.com>

jefftriplett commented on bylaws.md:
```diff
@@ -88,7 +88,7 @@ c) The corporation adopts and implements some means of verifying (1) that all pe
 ### SECTION 8. REGULAR AND ANNUAL MEETINGS ###
 
-Regular meetings of directors shall be held on the first Wednesday of each month at 2:00 PM, unless such day falls on a legal holiday, in which event the regular meeting shall be held at the same hour and place on the next business day.
+Regular meetings of directors shall be held on the first Wednesday of each month at 2:00 PM Central Time, unless such day falls on a legal holiday or a significant number of directors are unable to attend, in which event the regular meeting shall be held at the same hour and place on the next business day or another generally agreed upon day.
```
Historically we have changed our meeting cadence, and it's pure luck that we landed back on Wednesday @ 2 pm again. Maybe we should note that when the board changes, we poll the board for a monthly time that works for the group. We also reserve the right to not meet over the holidays or when a majority fo the board agrees we don't need to.

tim-schilling commented:
Need to update the annual meeting mention in the next paragraph.

tim-schilling committed:
Remove any mention of dates for meetings to match our practice.

tim-schilling committed:
Allow regular meetings to skip a month

This is to allow the board to decide to skip a meeting if there's nothing to discuss. This tends to be December.

tim-schilling commented:
Added the clause:

>  ..., unless the board votes to not hold a meeting.

jefftriplett commented on bylaws.md:
```diff
@@ -88,9 +88,9 @@ c) The corporation adopts and implements some means of verifying (1) that all pe
 ### SECTION 8. REGULAR AND ANNUAL MEETINGS ###
 
-Regular meetings of directors shall be held on the first Wednesday of each month at 2:00 PM, unless such day falls on a legal holiday, in which event the regular meeting shall be held at the same hour and place on the next business day.
+Regular meetings of directors shall be held regularly, at least once a month unless the board votes to not hold a meeting.
```
If we change it to quarterly, we can still meet monthly, and better reflect what we do without having to vote on not meeting. These tend to be last minute or in January when we all just want a break. 

```suggestion
Regular meetings of directors shall be held regularly, at least once a quarter, unless the board votes to not hold a meeting.
```

tim-schilling replied:
> If we change it to quarterly, we can still meet monthly, and better reflect what we do without having to vote on not meeting

How does it get decided when we want to meet or not meet?

Alternatively, it may be easier to go back to "Regular meetings of directors shall be held regularly once a month". If we don't meet quorum, we can't perform any actions anyway and it can't be counted against folks for not attending. The expectation / default should be to meet each month. If people don't show up then the meeting simply doesn't come to order?

jefftriplett replied:
> How does it get decided when we want to meet or not meet?

The president and/or secretary have historically set the cadence, polled the board, and we set the dates based on availability. If we have two or three directors say that can't make a meeting we have either rescheduled it or canceled it. That was historically because we wouldn't have a quorum so meeting wouldn't matter. 

Big picture: I think the goal here is to better reflect what we do without painting ourselves in a corner. I don't think we need to vote on not having a January meeting because it often falls on a date when most of the board is on vacation.

tim-schilling replied:
Fair enough, that works for me.

tim-schilling committed:
Update section 8 to state meet at least once a quarter.

Co-authored-by: Jeff Triplett <jefftriplett@users.noreply.github.com>

tim-schilling committed:
Replace mention march monthly meeting for first quarter meeting.
